common/psbt_open.c

```diff
@@ -153,7 +153,7 @@ static bool output_identical(const struct wally_psbt *a,
 static void sort_inputs(struct wally_psbt *psbt)
 {
 	/* Build an input map */
-	struct input_set *set = tal_arr(NULL,
+	struct input_set *set = tal_arr(tmpctx,
```

Comment:
nack, this is in common/ and tmpctx isn't passed in?

Reply:
Right right! thanks for this catch